pyproject.toml

```diff
@@ -43,19 +46,20 @@ watermark = "^2.4.3"
 
 [tool.pytest.ini_options]
 addopts = [
-    "-v",
-    "--strict-markers",
-    "--strict-config",
-    "--cov=pyfixest",
-    "--cov-report=term-missing",
-    "--color=yes",
+  "-v",
+  "--strict-markers",
+  "--strict-config",
+  "--cov=pyfixest",
+  "--cov-report=term-missing",
+  "--color=yes",
 ]
 markers = [
-    "extended: mark test to be part of the extended test suite"
+  "slow: mark test to be part of the slow test suite",
+  "extended: mark test to be part of the extended test suite",
```

Comment:
In the next iteration, we can split further by adding more markets if necessary.